FHIR-Converter/package.json

```diff
@@ -223,7 +223,7 @@
 		"vscode:prepublish": "npm run compile",
 		"compile": "tsc -b",
 		"watch": "tsc -b -w",
-		"postinstall": "cd client && npm install && cd ../server && npm install && cd ..",
+		"postinstall": "cd client && npm install && sh ./install-engine.sh && cd ../server && npm install && cd ..",
```

Comment:
When will this process be triggered?

Reply:
It will run automatically after we run `npm install`. Besides, we can trigger it using `npm run postinstall`.